kratos/includes/model_part.h

```diff
@@ -574,6 +605,106 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     }
 
 
+    ///@}
+    ///@name MasterSlaveConstraints
+    ///@{
+
+    SizeType NumberOfMasterSlaveConstraints() const
+    {
+        return mMasterSlaveConstraints.size();
+    }
+
+    MasterSlaveConstraintContainerType& MasterSlaveConstraints()
```

Comment:
make this const

Reply:
What if someone wants to loop over and change the constraint. !! The API does not exist now but in future ?